python/pyspark/ml/regression.py

```diff
@@ -2067,7 +2067,7 @@ def setTol(self, value):
         """
         return self._set(tol=value)
 
-    @since("2.2.0")
```

Comment:
This is a typo. should be 2.0.0.

Reply:
Right.
- https://spark.apache.org/docs/2.0.0/api/python/pyspark.ml.html#pyspark.ml.regression.GeneralizedLinearRegression.setFitIntercept